### ISSUE TITLE
Dirt rally tracking issue

### ISSUE THREAD

**Issue by CalamardoGuapo** [LibreVR/Revive#358]
Pimax 4k problem.
 Greetings, I have a problem with the pimax 4k and dirt rally. When I play and keep my head still, the game stops and a message comes out that I do not have the hmd connected (I suppose because it does not detect movement). Once you move your head, it detects movement again and removes the message, but it is very annoying to get you off the menu every time you keep your head still. I would like to know if you can solve this problem as it does not happen with Asseto corsa. Thank you.

**Comment by LibreVR** [comment]
This is not a tracking issue, the Oculus SDK has an API to query the headset proximity sensor to see if the user has put the headset on. Revive translates this to OpenVR "device activity level" calls, which on the Vive reports the whether the proximity sensor is triggered.

However since not many SteamVR developers use "device activity level" calls most third-party headset developers don't implement this function correctly. In your case the driver reports the device as "active" only if the headset is moving.

In other words, you should contact your headset vendor to fix the driver.

**Comment by flexodood** [comment]
it would be really nice if one could just override this fuction...
